packages/medusa/src/services/order-edit.ts

```diff
@@ -388,6 +411,86 @@ export default class OrderEditService extends TransactionBaseService {
     return orderEdit
   }
 
+  async addLineItem(orderEditId: string, data: AddOrderEditLineItemInput) {
+    return await this.atomicPhase_(async (manager) => {
+      const lineItemServiceTx = this.lineItemService_.withTransaction(manager)
+
+      const orderEdit = await this.retrieve(orderEditId, {
+        relations: [
+          "order",
+
+          "order.cart",
+          "order.cart.customer",
+          "order.cart.discounts",
+          "order.cart.discounts.rule",
+          "order.cart.gift_cards",
+          "order.cart.region",
+          "order.cart.region.tax_rates",
+          "order.cart.shipping_address",
+          "order.cart.shipping_methods",
+
+          "order.region",
+        ],
+      })
+
+      if (!OrderEditService.isOrderEditActive(orderEdit)) {
+        throw new MedusaError(
+          MedusaError.Types.NOT_ALLOWED,
+          `Can not add an item to the edit with status ${orderEdit.status}`
+        )
+      }
+
+      const regionId = orderEdit.order.region_id
+
+      // 0. check inventory
+
+      await this.inventoryService_
+        .withTransaction(manager)
+        .confirmInventory(data.variant_id, data.quantity)
```

Comment:
todo: remove this check. Since this is done by the store operator - who can view stock levels - we assume that they are aware of stock details and "give them full power".

We can always add this back in at a later point if relevant, but this is restricting things unnecessarily at this point.